components/salesforce/sources/new-object/new-object.js

```diff
@@ -0,0 +1,60 @@
+const startCase = require("lodash/startCase");
+
+const common = require("../../common");
+
+const EVENT_SOURCE_NAME = "New Object (Instant)";
+
+module.exports = {
+  ...common,
+  name: EVENT_SOURCE_NAME,
+  key: "salesforce-new-object",
+  description: "Triggers when a new object is created",
```

Comment:
Can you rephrase to the "Emits an event..." format?

Reply:
Done!